README.md

```diff
@@ -67,10 +67,10 @@ This repository uses the Kubernetes bots.  See a full list of the commands [here
 
 ### Implementer office hours
 
-Maintainers hold office hours every week, with sessions open to all developers
-working on this project.
+Maintainers hold office hours every fortnight, with sessions open to al
```

Comment:
```suggestion
Maintainers hold office hours every fortnight, with sessions open to all
```

Reply:
I'm not necessarily against the use of the word fortnight here, but I do wonder if it's not common enough terminology compared to `biweekly`

Reply:
The British have been eliminated.